recipes/level-zero/all/conanfile.py

```diff
@@ -0,0 +1,77 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.env import VirtualBuildEnv
+from conan.tools.cmake import CMakeToolchain, CMake, cmake_layout, CMakeDeps
+from conan.tools.files import apply_conandata_patches, get, export_conandata_patches
+
+class LevelZeroConan(ConanFile):
+    name = "level-zero"
+    package_type = "library"
+
+    license = "MIT"
+    homepage = "https://github.com/oneapi-src/level-zero"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "OneAPI Level Zero Specification Headers and Loader"
+    topics = ("api-headers", "loader", "level-zero", "oneapi")
+    package_id_non_embed_mode = "patch_mode"
+    package_type = "library"
+
+    # Binary configuration
+    settings = "os", "arch", "compiler", "build_type"
+    options =  {
+        "shared": [True, False],
```

Comment:
```suggestion
```

You may remove shared option, otherwise, when using -o "*/*:shared=False" will fail due the InvalidConfiguration in this recipe. In case no having the shared option, it will be linked to the shared library always.

Reply:
Hi, @uilianries, there's an error saying "Package type is 'library', but no 'shared' option declared", are you sure this option can be removed this case?